shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -2420,8 +2420,10 @@ export default {
           name="additionalmanifest"
           label-key="cluster.tabs.addOnAdditionalManifest"
           :showHeader="false"
+          @active="refreshComponentWithYamls('additionalmanifest')"
```

Comment:
i'm not 100% sure this is needed. yaml here refers to chart values, and additional manifest isn't one. was there a specific bug?